modular/executor/execute_gc.go

```diff
@@ -156,12 +157,32 @@ func (gc *GCWorker) checkGVGMatchSP(ctx context.Context, objectInfo *storagetype
 
 	if redundancyIndex == piecestore.PrimarySPRedundancyIndex {
 		if gvg.GetPrimarySpId() != spID {
+			swapInInfo, err := gc.e.baseApp.Consensus().QuerySwapInInfo(ctx, gvg.FamilyId, 0)
+			if err != nil {
+				if err == virtualgrouptypes.ErrSwapInInfoNotExist || err == virtualgrouptypes.ErrSwapInInfoExpired {
```

Comment:
Retruned err is a grpc error. There may be something wrong with the conditional judgment here.